### PR TITLE
[4.0] Label missing from some select filters

### PR DESCRIPTION
**Pull request by brianteeman**
Adds a label to the client, language and item type selectors in com_associations, com_menus and com_languages

### CODE REVIEW

File: administrator/components/com_associations/layouts/joomla/searchtools/default.php [hunk 1]
```diff
@@ -50,11 +50,13 @@
 	<?php // Add the itemtype and language selectors before the form filters. ?>
 	<div class="js-stools-container-selector-first">
 		<div class="js-stools-field-selector js-stools-itemtype">
+			<span class="sr-only"><?php echo $itemTypeField->label; ?></span>
```
Comment (Quy) [inline administrator/components/com_associations/layouts/joomla/searchtools/default.php]: In the xml file, add `labelclass="readonly"`, then the `span` tag can be removed.

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on 1f213c83f270282cafd15103098d30b2cced7ffe<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/24813">issues.joomla.org/tracker/joomla-cms/24813</a>.</sub>

**Comment by tonypartridge** [comment]
I have tested this item :white_check_mark: successfully on 1f213c83f270282cafd15103098d30b2cced7ffe<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/24813">issues.joomla.org/tracker/joomla-cms/24813</a>.</sub>

**Comment by Quy** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/24813">issues.joomla.org/tracker/joomla-cms/24813</a>.</sub>

**Comment by wilsonge** [comment]
Conflicts please

**Comment by brianteeman** [comment]
conflicts fixed

**Comment by wilsonge** [comment]
Thanks!

**Comment by brianteeman** [comment]
thanks

**Pull request merged**